app/views/staff/estates/show.html.haml

```diff
@@ -1,5 +1,10 @@
 - content_for :page_title_prefix, I18n.t('page_title.staff.estates.show', name: @estate.name)
 
+.govuk-breadcrumbs
+  %ol.govuk-breadcrumbs__list
+    = breadcrumb_link_to("Home", root_path)
+    = breadcrumb_current("Estate details for: #{@estate.name}")
```

Comment:
We should probably decide upon a convention for how we name our breadcrumbs so we keep it consistent for us and the users across all pages.

I have been using this convention:
```
"Scheme"#{scheme.name}"
"#{estate.name} estate"
"View scheme"#{scheme.name}"
```

Reply:
Would it actually make sense to use the same value as the page title for it so it always stays the same?

```
= breadcrumb_current("I18n.t('page_title.staff.estates.show', name: @estate.name)) 
```

Reply:
agree made the content more consistent